test/python/transpiler/test_sabre_layout.py

```diff
@@ -308,12 +307,10 @@ def test_support_var_with_explicit_routing_pass(self):
 
         cm = CouplingMap.from_line(8)
         with self.assertWarns(DeprecationWarning):
-            pass_ = SabreLayout(
-                cm, seed=0, routing_pass=StochasticSwap(cm, trials=1, seed=0, fake_run=True)
-            )
+            pass_ = SabreLayout(cm, seed=0, routing_pass=BasicSwap(cm, fake_run=True))
             _ = pass_(qc)
```

Comment:
This _shouldn't_ warn now... I think it might be the duration thing inside the implicit `dag_to_circuit` from `SabreLayout.__call__`.